R/stan_betareg.fit.R

```diff
@@ -20,7 +20,7 @@
 
 stan_betareg.fit <- function (x, y, z = NULL, weights = rep(1, NROW(x)), offset = rep(0, NROW(x)),
                               link = c("logit", "probit", "cloglog", "cauchit", "log", "loglog"), 
-                              link.phi = c("log", "identity", "sqrt"), ...,
+                              link.phi = "log", ...,
```

Comment:
I'm going through and making a few tweaks now. I can take care of this while I'm at it